cockatrice/src/filtertree.cpp

```diff
@@ -231,7 +231,22 @@ bool FilterItem::acceptManaCost(const CardInfoPtr info) const
 
 bool FilterItem::acceptCmc(const CardInfoPtr info) const
 {
-    return relationCheck(info->getCmc().toInt());
+    bool convertSuccess;
+    int cmcInt = info->getCmc().toInt(&convertSuccess);
+    // if conversion failed, check for the "//" separator used in split cards
+    if (!convertSuccess) {
+        int cmcSum = 0;
+        foreach (QString cmc, info->getCmc().split("//")) {
```

Comment:
can you change this to `for(QString cmc : info->getCMC().split("//")) {` ? I'm trying to remove the foreach from the codebase as it's not proper C++

Reply:
sure

Reply:
We should see if we can add a custom lint rule to codacy